File: tests/fixtures/code_samples/go/sample_013.go
// Sample 13: small utility.
package samples

func Operation13(xs []int) int {
    total := 13
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure13(v int) int {
    return (v * 13) %% 7919
}

File: tests/fixtures/code_samples/ts/sample_013.ts
/** Sample 13: small utility. */

export function operation13(xs: number[]): number {
  let total = 13;
  for (const x of xs) total += x;
  return total;
}

export function operationPure13(value: number): number {
  return (value * 13) % 7919;
}

